### PR TITLE
Add tweaks and changes

### PR DESCRIPTION
**Pull request by NolanK**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1007,10 +1007,6 @@ This API method is provided for convenience though may not serve a purpose in fu
 | ---| ---| ---| --- |
 | user\_id | string | QEaakM0qumTJd5tj | The Betable user\_id as returned from the [Create User](#create-user) method call. |
 
-### Request Parameters
-
-No request payload.
-
```
Comment (NolanK) [inline source/index.html.md:1013]: No Request parameters section needed for GET requests (same as health check above)

File: source/index.html.md [hunk 2]
```diff
@@ -448,17 +449,16 @@ curl \
...
+  "game_id": "ePz8u6hbncrNzFRCqLbX8e"
 }' https://ns-api-staging.betable.com/lfs/v100\
 /users/QEaakM0qumTJd5tj/rounds
 ```
 `POST /users/{user_id}/rounds`
 
 This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.
 
-One user may share many wager and payout events as long as the game_id and user_id remain the same.
+One round may share many wager and payout events as long as the game_id and user_id remain the same.
```
Comment (NolanK) [inline source/index.html.md:461]: As Alan pointed out, this line is not clear. Did we mean a "round" here? Also, what does this even mean?

File: source/index.html.md [hunk 3]
```diff
@@ -147,6 +147,12 @@ HTTP Status Code | Description
 }
 ```
 
+For each non-`2XX` status code returned, a JSON body will be returned to provide more context about the error encountered.
+
+Name | Type | Description
+ ---| ---| ---
+error\_description | string | A description of the error encountered; see [common errors](#common-error-responses) below or each API method section.
+
```
Comment (NolanK) [inline source/index.html.md:155]: This section was not formatted correctly with where it previously existed

File: source/index.html.md [hunk 4]
```diff
@@ -246,13 +247,12 @@ curl \
       "affiliate_program_id": "adwords",
```
Comment (ShawnSidoruk) [inline source/index.html.md:247]: Good call on removing the accepted terms.  I don't recall if there was a reason why "affiliate_details" is an array - that seems odd.

File: source/index.html.md [hunk 5]
```diff
@@ -298,7 +298,7 @@ A successful request returns the HTTP `201 Created` status code with the followi
 | user\_id | string | QEaakM0qumTJd5tj | The Betable user\_id as returned from the [Create User](#create-user) method call. |
 
 <aside class="success">
-The <code>user_id</code> returned should be used in subsequent API calls where a <code>user_id</code> is required as part of the URI.
+This <code>user_id</code> is used in subsequent calls when required as part of the URI.
```
Comment (NolanK) [inline source/index.html.md:301]: Making this consistent with the other asides like this one

File: source/index.html.md (hunk 2 above)
Comment (ShawnSidoruk) [reply source/index.html.md:461]: A round can arbitrarily have 1 wager or payout event or many of them.  The only limiting factor is that a round must maintain the same game_id.  This wording can definitely be improved but I'm not sure how best to do that.

Comment (ShawnSidoruk) [reply source/index.html.md:461]: Maybe: 

A round can consist of zero or more Wagers, and zero or more Payouts.  The choice of how many Wagers and Payouts to include in a round is up to the integration provider.  
[aside]
Wagers and Payouts will be presented back to the player using the "game_name" provided by the Round.
[/aside]

**Review by ShawnSidoruk — APPROVED** [review]
I have 2 questions in my review.

File: source/index.html.md [hunk 6]
```diff
@@ -778,7 +778,7 @@ curl \
 
 `POST /users/{user_id}/rounds/{round_id}/payouts`
 
-This method is used for creating and authorizing a Payout such as when awarding winnings. When possible, a Payout should be associated to a Wager.
+This method is used for creating and authorizing a Payout, such as for awarding winnings to a player. When possible, a Payout should be associated to a Wager.
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
This method is used for creating and authorizing a Payout, such as when awarding winnings to a player. When possible, a Payout should be associated to a Wager.
```

File: source/index.html.md (hunk 4 above)
Comment (dennis-mah) [reply source/index.html.md:247]: yeah wasn't sure why it was an array. I was hesitant to even put affiliate details and comms preferences in here.

File: source/index.html.md [hunk 7]
```diff
@@ -1063,7 +1058,7 @@ In order to add funds to the user's wallet (instead of overwriting the player's
 This method SHOULD NOT be used for resolving a Payout. That is, if a player should be awarded winnings, then the Create Payout method should be used.
```
Comment (dennis-mah) [inline source/index.html.md]:
```suggestion
This method <i>should not</i> be used for resolving a Payout. That is, if a player should be awarded winnings, then the <a href="#create-payout">Create Payout</a> method should be used.
```

Sorry piggy backing off your PR here to fix something I rushed around yesterday.

File: source/index.html.md (hunk 4 above)
Comment (NolanK) [reply source/index.html.md:247]: I just changed the example to use an object (as defined in the request parameters)

File: source/index.html.md [hunk 8]
```diff
@@ -1060,10 +1055,10 @@ Set Balance method completely overwrites the player's balance.
 In order to add funds to the user's wallet (instead of overwriting the player's balance), a Get Balance method call would have to be performed first to retrieve the user's existing balance, followed by a Set Balance method call using the value returned from Get Balance, adding the appropriate amount to the value returned from the user's current balance.
 
 <aside class="warning">
-This method SHOULD NOT be used for resolving a Payout. That is, if a player should be awarded winnings, then the Create Payout method should be used.
+This method <i>should not</i> be used for resolving a Payout. That is, if a player should be awarded winnings, then the <a href="#create-payout">Create Payout</a> method should be used.
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
This method <b><i>should not</i></b> be used for resolving a Payout. That is, if a player should be awarded winnings, then the <a href="#create-payout">Create Payout</a> method should be used.
```

**Pull request merged**